lldb/test/API/functionalities/fork/concurrent_vfork/TestConcurrentVFork.py

```diff
@@ -0,0 +1,49 @@
+"""
+Make sure that the concurrent vfork() from multiple threads works correctly.
+"""
+
+
+import lldb
+import lldbsuite.test.lldbutil as lldbutil
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import *
+
+
+class TestConcurrentVFork(TestBase):
+    NO_DEBUG_INFO_TESTCASE = True
+
+    def get_pid_from_variable(self):
+        target = self.dbg.GetTargetAtIndex(0)
+        return target.FindFirstGlobalVariable("g_pid").GetValueAsUnsigned()
+
+    @skipIfWindows
+    def test_vfork_follow_parent(self):
```

Comment:
We need to test this with exec and without exec to make sure things work without asserts firing from decrementing the fork count. So this function should be:
```
def vfork_follow_parent(self, call_exec):
```
Then we wall this twice:
```
def test_vfork_follow_parent_no_exec(self):
    return self.vfork_follow_parent(False);

def test_vfork_follow_parent_with_exec(self):
    return self.vfork_follow_parent(True);
```